### DISCUSSION TITLE
Welcome to Jellyman Discussions!

### DISCUSSION THREAD

**Discussion by Smiley-McSmiles** [Smiley-McSmiles/jellyman#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChevyTahoe666** [comment]
Has anyone been able to get Jellyman working on a Linux Mint version of Jellyfin installed prior to Linux Mint 21?

Linux Mint used a non-standard directory structure were things are distributed in /var/lib/jellyfin , var/log/jellyfin , var/cache/jellyfin, /usr/lib/jellyfin  etc.     

I am trying to install jellyman so I can back-up my previous install and transfer it to a fresh build.

 Any guidance would be greatly appreciated.....

**Reply by Smiley-McSmiles** [reply]
So Jellyman does not install along-side a current Jellyfin install. Jellyman installs then manages Jellyfin using the Jellyfin.tar.gz packages provided by the Jellyfin repo. It uses the generic Linux packages. This means that whatever method used to install Jellyfin will not be compatible with Jellyman. It is possible to migrate your current install of Jellyfin to how Jellyman needs it to be. But this is highly technical and it is generally recommended to just start from scratch with Jellyman. That way you can back it up and restore on any machine afterwards as long as you use Jellyman. It should be noted that Jellyman installs Jellyfin the way that the Jellyfin documentation asks. (If you decide to look up how to migrate using Jellyman.) If you have any other questions, feel free to ask. :) There is an example directory structure in the README.md. You could use this for an example on how to migrate to the generic Jellyfin type install. Again this is not recommended.

**Comment by ChevyTahoe666** [comment]
Thank Ypu for the quick response.

I was able to build the directory structure by creating links to the non-standard locations used by the mint installer and was finally able to get jellyfin to load and recognize my prior libraries, etc. ....  

If the backup/restore on the new machine does not work I will have to do a full fresh build..... But that will be a significant amount of work....

Thanks Again.....   Brian
